pkg/api/v1beta2/types.go

```diff
@@ -1094,3 +1102,23 @@ type ResourceQuotaList struct {
 	// Items is a list of ResourceQuota objects
 	Items []ResourceQuota `json:"items"`
 }
+
+// Secret holds secret data of a certain type
+type Secret struct {
+	TypeMeta `json:",inline"`
+
+	Data map[string][]byte `json:"data,omitempty"`
+	Type SecretType        `json:"type,omitempty"`
+}
+
+type SecretType string
+
+const (
+	SecretTypeOpaque SecretType = "opaque" // Default; arbitrary user-defined data
```

Comment:
@bgrant0607 I hadn't read it prior to your comment, but it's not clear.  There's no treatment given to constants or casing (viz: 'CamelCase' and 'constant' don't appear in the text of the doc).  I'll make an issue to clarify.